refinery/user_files_manager/urls.py

```diff
@@ -6,11 +6,5 @@
 
 user_files_router = DefaultRouter()
 user_files_router.urls.extend([
-    url(r'^files/$', UserFiles.as_view())  # Rooted under /api/v2
+    url(r'^files/$', UserFiles.as_view())
 ])
-user_files_url = url(r'^files/$',
```

Comment:
Are these unused? They are still being imported above in config/urls

Reply:
Good catch, I'll update import statements before merging. I wonder why didn't flake8 pick this up.